media-libs/libpng/libpng-1.5.26.recipe

```diff
@@ -2,16 +2,16 @@ SUMMARY="Portable Network Graphics library"
 DESCRIPTION="libpng is the official PNG reference library. It supports almost \
 all PNG features, is extensible, and has been extensively tested for over 17 \
 years"
-HOMEPAGE="http://www.libpng.org"
-COPYRIGHT="1998-2011 Glenn Randers-Pehrson
+HOMEPAGE="http://www.libpng.org/"
+COPYRIGHT="1998-2015 Glenn Randers-Pehrson
 	   1996-1997 Andreas Dilger
 	   1995-1996 Guy Eric Schalnat, Group 42, Inc."
 LICENSE="LibPNG"
-REVISION="2"
-SOURCE_URI="http://prdownloads.sourceforge.net/libpng/libpng-$portVersion.tar.gz"
-CHECKSUM_SHA256="1e25d7937ecda0bad46ab1b70f6f906c86efefc365a6f6c97ae6ae3895b17aa3"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/project/libpng/libpng15/$portVersion/libpng-$portVersion.tar.xz"
```

Comment:
prdownloads producde (at least) one extra redirect than with the "full" links. I have also noticed that the prdownloads links did not allways work in the past. And I beleive they have been deprecated (although they still work) in favor of the full links. But I can switch back to prdowloads if there is consensus for using short links.